diff --git a/lib/plugin/npm/npm.js b/lib/plugin/npm/npm.js
index 0e7caa39ba1311ea2530ddb01eb6522d50767fb2..af577d0b8df768e62a1307bb6ec2a0c03f20dce3 100644
--- a/lib/plugin/npm/npm.js
+++ b/lib/plugin/npm/npm.js
@@ -252,7 +252,7 @@ class npm extends Plugin {
       return false;
     }
     const args = [publishPath, `--tag ${tag}`, otpArg, dryRunArg, registryArg, ...fixArgs(publishArgs)].filter(Boolean);
-    return this.exec(`npm publish ${args.join(' ')}`, { options })
+    return this.exec(`pnpm publish ${args.join(' ')}`, { options })
       .then(() => {
         this.setContext({ isReleased: true });
       })
